src/front/ticketer/lib/backend_communication/model/event.dart

```diff
@@ -1,15 +1,14 @@
+import 'package:ticketer/backend_communication/model/event_status.dart';
 import 'package:ticketer/backend_communication/model/sector.dart';
 
-import 'event_status.dart';
-
 class Event {
   String? id;
   String name;
   String description;
   String location;
   String date;
   List<Sector> sectors;
-  EventStatus status;
+  EventStatus? status;
```

Comment:
Why this can be null? Add explorer comment that this is done in order to facilitate noStatus events